### ISSUE TITLE
Language server build fails - coqc not available?

### ISSUE THREAD

**Issue by redanaheim** [coq-community/vscoq#747]
Running `nix build github:coq-community/vscoq` which by default builds the language server with coq 8.19:
```
       >Running phase: unpackPhase
       > unpacking source archive /nix/store/n44kprd8pzy0c1r4q1kiqmpy0l2zv35y-language-server
       > source root is language-server
       > Running phase: patchPhase
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > no configure script, doing nothing
       > Running phase: buildPhase
       > Error: Program coqc not found in the tree or in PATH
       >  (context: default)
       > -> required by %{coq:version.major} at dm/dune:5
       > -> required by _build/default/dm/.merlin-conf/lib-vscoq-language-server.dm
       > -> required by _build/default/dm/dm.cmxa
       > -> required by _build/default/vscoqtop/vscoqtop.exe
       > -> required by _build/install/default/bin/vscoqtop
       > -> required by _build/default/vscoq-language-server.install
       > -> required by alias install
```

(using nix version 2.20.3 on macOS)

Building the release 2.0.3+coq8.18 works fine. Also, if I clone the repo, adding `coq_8_19` to the `nativeBuildInputs` in the relevant package in `flake.nix` (it's currently only in `buildInputs`) the language server builds correctly.

I don't know enough about nix or this specific project yet to know for sure, but it seems to me like maybe `coqc` was not required for the build process of this package at the time of the 2.0.3 release, and it is required now, but somebody forgot to add it to the `nativeBuildInputs` so that it was available.

**Comment by rtetley** [comment]
Hey ! Thanks for reporting ! If you already have a fix in a fork, could you open a pull request ? Would be awesome :-)

**Comment by redanaheim** [comment]
Alright sure, I do believe I have a fix. https://github.com/coq-community/vscoq/pull/754

**Comment by rtetley** [comment]
Closed by #754
Thanks !
